qualtran/bloqs/chemistry/sparse/sparse_test.py

```diff
@@ -125,11 +77,8 @@ def test_sparse_costs_against_openfermion(num_spin_orb, num_bits_rot_aa):
     )
     # correct for SelectSwapQROM vs QROAM
     # https://github.com/quantumlib/Qualtran/issues/574
-    paper_qrom = qrom_cost(prep_sparse)
-    paper_qrom += qrom_cost(prep_sparse_adj)
-    qual_qrom_cost = get_sel_swap_qrom_toff_count(prep_sparse)
-    qual_qrom_cost += get_sel_swap_qrom_toff_count(prep_sparse_adj)
-    delta_qrom = qual_qrom_cost - paper_qrom
+    # The -2 comes from a more accurate calculation of the QROAM costs in Qualtran.
+    delta_qrom = -2
```

Comment:
Feel free to elaborate -- "The -2 comes from a more accurate calculation of the QROAM costs in Qualtran. The difference arises from uncontrolled unary iteration used by QROM, which QROAMClean delegates to. "